src/js/tabs/pid_tuning.js

```diff
@@ -1394,6 +1394,10 @@ TABS.pid_tuning.initialize = function (callback) {
             if (semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_39)) {
                 filterTypeValues.push("FIR");
             }
+            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+                filterTypeValues.push("PT2");
+                filterTypeValues.push("PT3");
+            }
```

Comment:
I don't know if I like more this way or only with one push...
```suggestion
            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
                filterTypeValues.push("PT2", "PT3");
            }
```

Reply:
Interesting, but maybe i like the first way to best visual differentation betwen two filters type. Do you think that with a single push the performance will be better?

Reply:
I'm not too sure what I prefer, only was a comment 🙃

Reply:
I prefer the former as it's easier to understand :upside_down_face: